R/India.R

```diff
@@ -37,7 +37,7 @@ India <- R6::R6Class("India",
     source_url = "https://www.covid19india.org",
 
     #' @description Set up a table of region codes for clean data
-    #' @importFrom tibble tibble
+    #' @importFrom dplyr tibble
```

Comment:
Is this (importing from dplyr vs the package it comes from recommended practice?) I have always been a bit worried about it as it feels like masking dependencies but perhaps that is a mistaken notation.

Reply:
I couldn't find any official documentation confirming this. Is the fact that this is used inside tidyverse packages a good argument?

https://github.com/tidymodels/broom/blob/5057297a32811544f225d7d0a079a9d9cd8c8633/R/margins-tidiers.R#L86

Reply:
I think it is a good enough one for sure. Another little rOpenSci post 😉